src/firefly/js/templates/lightcurve/LcResult.jsx

```diff
@@ -115,6 +116,28 @@ export class LcResult extends PureComponent {
     }
 }
 
+export function defaultDownloadPanel(mission='', cutoutSize, addtlParams={}) {
+    mission = mission.replace(/[\/ ]/g, '_');       // clean up mission description to be used for save as value.
+    return (
+        <DownloadButton>
+            <DownloadOptionPanel
+                groupKey = {mission}
+                dataTag = {DL_DATA_TAG}
+                cutoutSize={cutoutSize}
+                title={'Image Download Options'}
+                dlParams={{
+                    MaxBundleSize: 200 * 1024 * 1024,    // set it to 200mb to make it easier to test multi-parts download.  each wise image is ~64mb
+                    TitlePrefix: mission,
```

Comment:
Title needs to have date and time.  That was a ticket a year ago to request that kind format.  But I can add it after you merge to the dev.

Reply:
Check with @ejoliet before putting it back.  If I understand it correctly, it was requested because there was only one field used to hold both Title and Save As.  I've now separate them.  It should be clearer now without having to introduce timestamps into the label.

Reply:
Yes, the label is fine and should be fixed (even though CCB could argue to change it to something else in the future) but the value is the one you can pass outside and this one will contain time stamp. So i think we are good. @lznakano should confirm once you merge.